README.md

```diff
@@ -1,3 +1,114 @@
 # Node Debugger
 
 ## Overview
+
+This lesson teaches how to use the built-in Node CLI debugger.
+
+## Objectives
+
+1. Describe how to start Node debugger
+1. Describe debugger and the commands
+
+## Describe how to start Node debugger
+
+Have you ever had a pesky bug that you couldn't track with simple print or log statements? Console.log (print statements in JavaScript/Node) are okay until they are not enough. Sometimes, it's painful not to be able to stop the execution process to look around the scope and investigate further. Node comes with a built-in debugger. All you need to do is to start the program in a debug mode:
+
+```
+$ node debug program.js
+```
+
+The prompt will turn into `debug>`. You can get the list of command with the `help` command.
+
+## Describe debugger and the commands
+
+The debugger commands won't be new to a person familiar with debugging in other languages like Ruby or browser JavaScript (with the help of DevTools). There are:
+
+* run: run program
+* cont (`c`): proceed with the execution
+* next (s): next step
```

Comment:
would be great to see examples of what step in and step out do, as well as sb and cb